src/Symfony/Component/Routing/Route.php

```diff
@@ -70,6 +70,7 @@ class Route implements \Serializable
      * Available options:
      *
      *  * compiler_class: A class name able to compile this route instance (RouteCompiler by default)
+     *  * utf8:           Whether UTF-8 matching is enforced ot not
```

Comment:
utf8 is now an option on the route. ping @stof (I don't know were I've got this idea that it wasn't possible...)